test/contracts/promises/three-contracts/first-contract/src/lib.rs

```diff
@@ -0,0 +1,128 @@
+#![no_std]
+#![no_main]
+#![allow(unused_attributes)]
```

Comment:
No need for this, there are no attributes here.

Reply:
Removed.